api/model/src/main/resources/org/projectnessie/model/NessieConfiguration.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
File's intended for two things:
* Define the versions in a text file vs code
* External tools can extract this piece of information from the nessie-model.jar